geps/gep-713/index.md

```diff
@@ -26,16 +20,25 @@ settings across either one object (this is "Direct Policy Attachment"), or objec
 in a hierarchy (this is "Inherited Policy Attachment").
 
 Individual policy APIs:
-- must be their own CRDs (e.g. `TimeoutPolicy`, `RetryPolicy` etc),
-- can be included in the Gateway API group and installation or be defined by
+- MUST be their own CRDs (e.g. `TimeoutPolicy`, `RetryPolicy` etc),
+- MUST include both `spec` and `status` stanzas
+- MUST have the `status` stanza include a `conditions` section using the standard
+  upstream Condition type
+- MAY be included in the Gateway API group and installation or be defined by
   implementations
-- and must include a common `TargetRef` struct in their specification to identify
+- MUST include a common `TargetRef` struct in their specification to identify
   how and where to apply that policy.
-- _may_ include either a `defaults` section, an `overrides` section, or both. If
-  these are included, the Policy is an Inherited Policy, and should use the
-  inheritance rules defined in this document.
+- MAY affect more objects than the object specified in the `targetRef`. In this
+  case, the Policy is an Inherited Policy. A common way to do this is to include
+  either a `defaults` section, an `overrides` section, or both.
+- Policy objects that affect _only_ the object specified in the `targetRef` are
```

Comment:
Can there be more than one object specified in the targetRef, as long as they are not hierarchically related?
```suggestion
- Policy objects that affect _only_ the object/s specified in the `targetRef` are
```

Reply:
As of right now, no. I'm open to discussing that, but I think that should be a separate follow-up discussion (pretty sure that @guicassolato and @arkodg want multiple `targetRef`s in one Policy to be given the thumbs-up).

Reply:
yah would love guidance / nod reg multiple `targetRefs` from GatewayAPI
cc @guydc

Reply:
Istio has policies which allow specifying either a `targetRef` or `selector` such as https://istio.io/latest/docs/reference/config/security/authorization-policy/#AuthorizationPolicy - I could see plural `targetRefs` being useful too.

Reply:
:+1: to `targetRefs`